spec/paper_trail/config_spec.rb

```diff
@@ -32,6 +32,14 @@ module PaperTrail
       end
     end
 
+    # describe "association_reify_error_behaviour" do
+    #   it "accepts a value" do
+    #     config = described_class.instance
+    #     config.association_reify_error_behaviour = :warn
+    #     expect(config.association_reify_error_behaviour).to eq(:warn)
+    #   end
+    # end
```

Comment:
Given the other specs you've added, it looks like we can delete this commented-out spec?